src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseSearchEndpoint.java

```diff
@@ -87,25 +73,6 @@ public CaseSearchResult searchCases(
         return result;
     }
 
-    private JsonNode stringToJsonNode(String jsonSearchRequest) {
-        return objectMapperService.convertStringToObject(jsonSearchRequest, JsonNode.class);
-    }
-
-    private void rejectBlackListedQuery(String jsonSearchRequest) {
-        List<String> blackListedQueries = applicationParams.getSearchBlackList();
-        Optional<String> blackListedQueryOpt = blackListedQueries
-            .stream()
-            .filter(blacklisted -> {
-                Pattern p = Pattern.compile("\\b" + blacklisted + "\\b");
-                Matcher m = p.matcher(jsonSearchRequest);
-                return m.find();
-            })
-            .findFirst();
-        blackListedQueryOpt.ifPresent(blacklisted -> {
-            throw new BadSearchRequest(String.format("Query of type '%s' is not allowed", blacklisted));
-        });
-    }
-
     public static String buildCaseIds(CaseSearchResult caseSearchResult) {
```

Comment:
my IDE flag this as unused. Can we remove?

Reply:
This is still required (reference by string) for log and audit.